runtime/src/main/kotlin/kotlin/text/Strings.kt

```diff
@@ -7,6 +7,185 @@ package kotlin.text
 
 import kotlin.comparisons.*
 
+/**
+ * Returns the index within this string of the first occurrence of the specified character, starting from the specified offset.
+ */
+@SymbolName("Kotlin_String_indexOfChar")
+external internal actual inline fun String.nativeIndexOf(ch: Char, fromIndex: Int): Int
```

Comment:
`external inline` — I wonder does it make sense?

Reply:
Unlikely so.

Reply:
I've found why they are `inline` — because they are `inline` in the corresponding expect declaration. I'll remove it there and then here.